### ISSUE TITLE
Affix disappearing when scrolling on Android

### ISSUE THREAD

**Issue by riwu** [stephane-monnot/react-vertical-timeline#116]
https://i.imgur.com/MY8DRcC.mp4
https://6rkwm.csb.app/
https://codesandbox.io/s/basic-antd4167-forked-6rkwm?file=/index.js

On Android Chrome, the affix is disappearing when you scroll (see video).
Works fine on PC/iPhone.

**Comment by RyanGosden** [comment]
I have the same issue with my navigation menu. Looking into finding a solution.

**Comment by RyanGosden** [comment]
> https://i.imgur.com/MY8DRcC.mp4
> https://6rkwm.csb.app/
> https://codesandbox.io/s/basic-antd4167-forked-6rkwm?file=/index.js
> 
> On Android Chrome, the affix is disappearing when you scroll (see video).
> Works fine on PC/iPhone.

Its a temporary solution, but it does not occur when animate is set to false.

**Comment by riwu** [comment]
> Its a temporary solution, but it does not occur when animate is set to false.
Yeah that's what I'm currently doing for Android, but hoping for a better fix.
